web/themes/custom/recipe_theme/includes/page.inc

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @file
+ * Theme and preprocess functions for pages.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Template\Attribute;
+
+/**
+ * Implements hook_preprocess_HOOK() for page.html.twig.
+ */
+function recipe_theme_preprocess_page(array &$variables): void {
+  $variables['page_attributes'] = new Attribute();
+  $route_name = \Drupal::routeMatch()->getRouteName();
+  if ($route_name === 'entity.user.canonical') {
+    $variables['is_userpage'] = TRUE;
```

Comment:
This is a nice solution, but a little overkill. Please see my other comment at https://github.com/Pronovix/food/pull/45/files#diff-b2f684ce1ef5fd62d13500f925ecb33bR9